src/issue/screens/new-issue.screen.js

```diff
@@ -90,17 +92,19 @@ class NewIssue extends Component {
       ).then(issue => {
         navigation.navigate('Issue', {
           issue,
+          headerLeft: false,
```

Comment:
I can try to implement the 1) (with `gesturesEnabled: false` + `headerLeft: null`) 😄 

But I don't understand the number 2 with modal :(